docs/guides/save_tf_state_in_vkcs_s3.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: "vkcs"
+page_title: "Save Terraform state in VKCS S3"
+description: |-
+  Save Terraform state in VKCS Cloud Storage
+---
+
+## Configure a bucket
+The first thing you need is to [create and manage an account](https://cloud.vk.com/docs/en/storage/s3/service-management/account-management).
```

Comment:
account in Cloud Storage?

and write smth about creds. i realize there is no appropriate doc but this should be mentioned

Reply:
fix link annotation, but I don't understand what information is needed about creds